Mage/src/main/java/mage/abilities/dynamicvalue/common/PermanentsSacrificedThisTurnCount.java

```diff
@@ -0,0 +1,40 @@
+
+package mage.abilities.dynamicvalue.common;
+
+import mage.abilities.Ability;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.effects.Effect;
+import mage.game.Game;
+import mage.watchers.common.PermanentsSacrificedWatcher;
+
+/**
+ * @author Susucr
+ */
+public enum PermanentsSacrificedThisTurnCount implements DynamicValue {
+    instance;
+
+    @Override
+    public int calculate(Game game, Ability sourceAbility, Effect effect) {
+        PermanentsSacrificedWatcher watcher = game.getState().getWatcher(PermanentsSacrificedWatcher.class);
+        if (watcher != null) {
+            return watcher.getThisTurnSacrificedPermanents();
+        }
+        return 0;
+    }
+
+    @Override
+    public PermanentsSacrificedThisTurnCount copy() {
+        return PermanentsSacrificedThisTurnCount.instance;
+    }
+
+    @Override
+    public String toString() {
+        return "X";
```

Comment:
not sure if it's possible to adjust this to get the text generation to work as "for each"? I haven't investigated how dynamic value text works and a lot of existing cards do have to set text manually, so don't need to worry about it for this PR

Reply:
I am really unsure, most of the `DynamicValue` do have "X" as that `toString` and I did not give it more thought.